client/my-sites/checkout/index.js

```diff
@@ -64,6 +65,7 @@ export default function () {
 
 	page(
 		'/checkout/thank-you/:site/:receiptId?',
+		hideTheMasterbar,
```

Comment:
I tried not to reinvent the wheel with this hideTheMasterbar function.

I found the const noMasterbarForRoute and noMasterbarForSection in 'client/layout/index.jsx.' I could have used those. However, many checkout pages share the same 'checkout-thank-you' section name and regex on our checkout route path feels prone for errors.

The approach I took with the hideTheMasterbar function allow us to be explicit as to which route we wish to hide the masterbar. But, I wonder if it would be suited somewhere more globally available in scope? It seems like a function such as this might be reused over multiple routes outside of checkout. I did look, but maybe I missed an existing method?